packages/workbox-google-analytics/test/static/default.js

```diff
@@ -0,0 +1,57 @@
+/*
+ Copyright 2017 Google Inc. All Rights Reserved.
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+ http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+importScripts('/node_modules/sinon/pkg/sinon.js');
+importScripts('/__test/bundle/workbox-google-analytics');
+
+self.addEventListener('install', () => self.skipWaiting());
+self.addEventListener('activate', (evt) => evt.waitUntil(clients.claim()));
+
+// Add a sync event prior to calling googleAnalytics.initialize(), so this
+// listener runs before any other sync listeners.
+self.addEventListener('sync', (evt) => {
```

Comment:
Why are you setting a sync here at all?

Reply:
I need to unstub (and then restub) `fetch` in the service worker before the background sync queue is retried.

Since I know a `sync` event will happen before the queue is retried, listening for that was the easiest way.

Reply:
Could you extend the comment to explain this.